diff -Naur csv9.5-src/makefiles/Mf-install.in csv9.5/makefiles/Mf-install.in
--- csv9.5-src/makefiles/Mf-install.in	2017-10-11 23:00:29.000000000 +0200
+++ csv9.5/makefiles/Mf-install.in	2018-05-19 22:44:47.188070813 +0200
@@ -122,7 +122,7 @@
           /bin/rm -f ${LibBin}/${InstallSchemeName}.boot;\
           ln -f ${LibBin}/scheme.boot ${LibBin}/${InstallSchemeName}.boot;\
         fi
-	ln -sf ${LibBin}/scheme.boot ${LibBin}/${InstallScriptName}.boot;
+	ln -f ${LibBin}/scheme.boot ${LibBin}/${InstallScriptName}.boot;
 	$I -m 444 ${Include}/scheme.h ${LibBin}
 
 maninstall: scheme.1 petite.1 ${Man}

