diff -Naur csv9.5-src/c/expeditor.c csv9.5/c/expeditor.c
--- csv9.5-src/c/expeditor.c	2017-06-08 05:52:53.000000000 +0200
+++ csv9.5/c/expeditor.c	2018-05-19 19:19:18.233746893 +0200
@@ -546,7 +546,9 @@
 #include <sys/ioctl.h>
 #include <wchar.h>
 #include <locale.h>
+#ifdef I_XLOCALE
 #include <xlocale.h>
+#endif
 
 #if defined(TIOCGWINSZ) && defined(SIGWINCH) && defined(EINTR)
 #define HANDLE_SIGWINCH
